core/src/jsMain/kotlin/dev/fritz2/core/rendercontext.kt

```diff
@@ -39,7 +39,69 @@ interface RenderContext : WithJob, WithScope {
     }
 
     /**
-     * Renders the data of a [Flow] only if the [predicate] is true.
+     * Renders the data of a [Flow] only if the given [predicate] is true.
+     *
+     * Example:
+     * ```kotlin
+     * data class Delivery(
+     *     val state: String,
+     *     val visibility: Boolean
+     * )
+     *
+     * val storedDelivery = storeOf(Delivery("in production", true))
+     *
+     * storedDelivery.data.renderIf(Delivery::visibility) { delivery ->
+     *     // This whole block might be re-rendered if the state is changed! (read on)
+     *     p { +"The state of your delivery is ${delivery.state}" }
+     * }
+     * ```
+     *
+     * Beware that the data value [V] and its changes are still handled reactively inside the [content] expression!
+     *
+     * So while the [predicate] may remain stably `true`, some other aspect of the model [V] may change in between.
+     * Then the whole [content]-block will be re-rendered accordingly of course!
```

Comment:
> Then the whole [content]-block ...

Maybe use "As a result, " over "Then"?

> ... will be re-rendered ...

In this case I'd use "would" over "will" since you're using conditional case in the sentence before